examples/resources/incident_schedule/resource.tf

```diff
@@ -0,0 +1,76 @@
+data "incident_user" "rory" {
+  email = "rory.malcolm@incident.io"
+}
+
+data "incident_user" "martha" {
+  email = "martha@incident.io"
+}
+
+data "incident_user" "roryb" {
+  email = "rory@incident.io"
+}
+
+resource "incident_schedule" "testing_my_provider" {
+  name = "Testing Terraform"
+
+  // from tz database
```

Comment:
Shall we expand this? Presumably this is an ISO format we can reference?

Reply:
There's no ISO format afaik - `tz` is the scientific term! https://en.wikipedia.org/wiki/List_of_tz_database_time_zones